pkg/model/api/config/authentication.go

```diff
@@ -16,6 +16,7 @@ package config
 import (
 	"errors"
 	"fmt"
+	"github.com/prometheus/common/config"
```

Comment:
import not in the right place, shouldn't be mixed with native import

Reply:
New workstation, have to configure again my IDE hehe